phoenix-server/pom.xml

```diff
@@ -173,6 +173,7 @@
     <dependency>
       <groupId>org.apache.tephra</groupId>
       <artifactId>tephra-core</artifactId>
+      <version>${tephra.version}</version>
```

Comment:
The versions should be defined in the top project dependencyManagement section

(Why do we even have test dependencies in phoenix-server ? - that's a question for another day)